packages/discovery-styles/scss/components/search-facets/_search-facets.scss

```diff
@@ -10,6 +10,7 @@
 
 .#{$prefix}--search-facet {
   margin-bottom: $spacing-md;
+  word-break: break-word;
```

Comment:
If I'm reading the screenshots correctly, this will also break a very long word, correct?